src/core/src/validation_util.cpp

```diff
@@ -1188,6 +1188,15 @@ std::shared_ptr<ov::op::v0::Constant> ov::util::constantfold_subgraph(const Outp
     if (num_inputs == 0)
         return nullptr;
 
+    if (subgraph_sink.get_tensor().has_and_set_bound()) {
+        const auto& lower = subgraph_sink.get_tensor().get_lower_value();
+        return std::make_shared<ov::op::v0::Constant>(lower.get_element_type(), lower.get_shape(), lower.data());
```

Comment:
```suggestion
        return std::make_shared<ov::op::v0::Constant>(lower);
```
Will it work? If yes there will be no copy of tensor data.